### ISSUE TITLE
Add printer emulation

### ISSUE THREAD

**Issue by robinhood2014** [mauiaaron/apple2#31]
I would love the ability to print to files like I can on the parent project AppleWin (e.g. PR#1, LIST, PR#0).

**Comment by mauiaaron** [comment]
Agreed, this would be a very nice feature to have both for Android and the other ports.
